rules/filename-case.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+var path = require('path');
```

Comment:
Add an empty line below.

Should we maybe have a rule in the `import` plugin to enforce this?